### PR TITLE
Updating openapi spec for function calling

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +44,11 @@ Here are the steps required to publish a new version of our API clients.
 1. cd into the `django` directory from within the [main vellum repo](https://github.com/vellum-ai/vellum)
 2. Run `make generate-swagger-api-client`
 3. Open the generated `schema_api_client_v1.yaml` file
-4. Copy the contents of the file and paste it into the definition within this repo: [fern/api/openai/openapi.yaml](./fern/api/openai/openapi.yaml)
+4. Copy the contents of the file and paste it into the definition within this repo: [fern/api/openapi/openapi.yaml](./fern/api/openapi/openapi.yaml)
 5. Check the diff and selectively add everything except for deletions to `server`. These were manually added.
     Note: We should automate the step of adding `server` related info back as part of `make generate-swagger-api-client`.
 6. Run `fern check` to make sure all is good.
 7. Optionally run `fern generate` to see the newly generated clients locally.
+    - If this is your first time, run hit `Y` to logging in and fern will log you in through GitHub.
```
Comment (dvargas92495) [inline README.md:52]: even after logging in, I got this error: `[api]: No group specified. Use the --group option, or set "default-group" in generators.yml`

File: fern/api/openapi/openapi.yml [hunk 2]
```diff
@@ -739,15 +756,29 @@ components:
         status:
           allOf:
           - $ref: '#/components/schemas/DeploymentReadStatusEnum'
-          description: The current status of the deployment
+          description: |-
```
Comment (dvargas92495) [inline fern/api/openapi/openapi.yml:762]: I assume a lot of these are just from the API client not having had updated in a bit

File: README.md (hunk 1 above)
Comment (noanflaherty) [reply README.md:52]: Grammar mistakes here

File: fern/api/openapi/openapi.yml [hunk 3]
```diff
@@ -680,16 +680,22 @@ paths:
                 $ref: '#/components/schemas/UploadDocumentErrorResponse'
           description: ''
       servers:
-        - url: https://documents.vellum.ai
-          x-name: Documents
+        - url: https://predict.vellum.ai
```
Comment (noanflaherty) [inline fern/api/openapi/openapi.yml:683]: I dont think we want to make this change

File: fern/api/openapi/openapi.yml (hunk 2 above)
Comment (noanflaherty) [reply fern/api/openapi/openapi.yml:762]: Yeah exactly

**Review by noanflaherty — APPROVED** [review]
Please see comments and fix prior to merging

File: fern/api/openapi/openapi.yml (hunk 3 above)
Comment (dvargas92495) [reply fern/api/openapi/openapi.yml:683]: oh whoops

**Pull request merged**